ios/fastlane/Fastfile

```diff
@@ -115,6 +124,20 @@ platform :ios do
       app_identifier: "#{generate_app_bundle_id}",
       plist_path: "Inji/Info.plist"
     )
+    # replace bundle ID
+    s = "s/mobileid/#{map_flavour_tosuffix}/g"
+    sh("sed -i '' -e #{s} ios/Inji.xcodeproj/project.pbxproj")
+
+    # replace Product name
+    p = "s|PRODUCT_NAME = Inji|PRODUCT_NAME = \"#{generate_app_name}\"|"
+    sh ("sed -i '' -e #{p} ios/Inji.xcodeproj/project.pbxproj")
+
+    # update app display name
+    update_info_plist(
+      plist_path: 'Inji/Info.plist',
+      display_name: "#{generate_app_name}"
+    )
```

Comment:
these might be accomplished via https://docs.fastlane.tools/actions/update_project_provisioning/ and can be taken for later refactoring IMO https://github.com/fastlane/fastlane/issues/8973 as we are short on time and blocking an internal release